### ISSUE TITLE
upgraded from 2.1.15 to 2.1.18 and can no longer build

### ISSUE THREAD

**Issue by dante-blitz** [datocms/gatsby-source-datocms#82]
Hi there,

I just upgraded to the latest version (2.1.18 from 2.1.15) and I get the following when running gatsby develop...

```
success run queries - 12.014s - 113/113 9.41/s
success run queries - 113/113 Infinity/s
```

The develop process never completes. Have reverted back to previous version.

---

  System:
    OS: Windows 10
    CPU: (8) x64 Intel(R) Xeon(R) CPU E5-1620 0 @ 3.60GHz
  Binaries:
    Yarn: 1.16.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.3 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 2.7.15 - /c/Users/user/.windows-build-tools/python27/python
  Browsers:
    Edge: 44.18362.387.0
  npmPackages:
    gatsby: ^2.16.4 => 2.16.4 
    gatsby-image: ^2.2.29 => 2.2.29 
    gatsby-plugin-eslint: ^2.0.5 => 2.0.5 
    gatsby-plugin-material-ui: ^2.1.6 => 2.1.6 
    gatsby-plugin-offline: ^3.0.16 => 3.0.16 
    gatsby-plugin-react-helmet: ^3.1.13 => 3.1.13 
    gatsby-plugin-react-svg: ^2.1.2 => 2.1.2 
    gatsby-plugin-sharp: ^2.2.32 => 2.2.32 
    gatsby-plugin-sitemap: ^2.2.19 => 2.2.19 
    gatsby-plugin-styled-components: ^3.1.11 => 3.1.11 
    gatsby-plugin-webpack-bundle-analyzer: ^1.0.5 => 1.0.5 
    gatsby-plugin-webpack-size: ^1.0.0 => 1.0.0 
    gatsby-source-datocms: 2.1.18 => 2.1.18 
    gatsby-transformer-remark: ^2.6.30 => 2.6.30 
    gatsby-transformer-sharp: ^2.2.23 => 2.2.23

**Comment by matjack1** [comment]
Hello @dante-blitz have you tried the usual `gatsby clean` and deleting the `node_modules` folder and reinstall all the dependencies?

If yes, can you please help me reproducing the error by sending me over the support form: https://www.datocms.com/support/ your project details? If it's a problem only of your account I'm going to close this issue.

**Comment by einomi** [comment]
I'm having a similar issue after uprade to `2.1.18`:
`TypeError: Cannot read property 'allDatoCmsCafeMenuCategory' of undefined`

I've removed .cache/ public/ and node_modules/, package-lock.json, reinstalled dependencies, tried `gatsby clean`, upgraded gatsby to the latest version but the error still appears. I'm going to roll back to the previous version for now.

**Comment by matjack1** [comment]
@einomi have you rolled back to 2.1.7 or 2.1.5?

**Comment by matjack1** [comment]
Closing now as we've released also v2.1.22.

Please reopen if it's still happening with latest releases. Thank you!

**Comment by einomi** [comment]
> @einomi have you rolled back to 2.1.7 or 2.1.5?
Yes, I think I did
